### PR TITLE
Add Yield+ on EOS

### PR DESCRIPTION
**Pull request by DenisCarriere**
GitHub repos for: https://tokenyield.io/ (Yield+ on EOS)

### CODE REVIEW

File: data/ecosystems/y/yield-eos.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Ecosystem Level Information
+title = "Yield+ on EOS"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/eosnetworkfoundation"]
```
Comment (eherrerosj) [inline data/ecosystems/y/yield-eos.toml:6]: EOS Network Foundation org is already part of Antelope, which is a child eco of EOS

File: data/ecosystems/y/yield-eos.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Repositories
+[[repo]]
+url = "https://github.com/eosnetworkfoundation/eosio.yield-contracts"
+
+[[repo]]
+url = "https://github.com/eosnetworkfoundation/eosio.yield-docs"
+
+[[repo]]
+url = "https://github.com/defiboxswap/eosio.yield-ui"
```
Comment (eherrerosj) [inline data/ecosystems/y/yield-eos.toml:16]: Let's keep this one only

**Review by eherrerosj — CHANGES_REQUESTED** [review]

File: data/ecosystems/y/yield-eos.toml (hunk 2 above)
Comment (eherrerosj) [reply data/ecosystems/y/yield-eos.toml:16]: hey @DenisCarriere, were you able to take a look at my last comment?

**Comment by eherrerosj** [comment]
Hi @DenisCarriere, we wanted to give you a heads-up that we are planning to freeze the data set that will be used for the analysis during the next few days. If you want your changes to be included and have an impact on this year’s report, please ensure that the requested modifications are made and merged prior to the data freeze.

Let's aim to have your PR finalized as soon as possible to make it into this window. If you have any questions or need further clarification, feel free to reply here or reach out directly.

**Comment by jubos** [comment]
Manually merged this with 2b39ff7

**Pull request closed without merging**